src/Simplex/Messaging/Server/Main.hs

```diff
@@ -626,6 +632,9 @@ defaultDBConnStr = "postgresql://smp@/smp_server_store"
 defaultDBSchema :: ByteString
 defaultDBSchema = "smp_server"
 
+defaultDBPoolSize :: Natural
+defaultDBPoolSize = 10
```

Comment:
why not more? they write maximum concurrent connections is 100 by default, I thought we'd configure 20 at least

Reply:
it's in INI file